bundle/config/mutator/resource_key.go

```diff
@@ -0,0 +1,106 @@
+package mutator
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/databricks/cli/libs/dyn"
+)
+
+// ResourceKey uniquely identifies a resource in configuration.
+type ResourceKey struct {
+	// Type is the type of the resource. E.g. "jobs", "dashboards", etc.
+	Type string
+
+	// Path is the path to the resource.
+	Name string
+}
+
+// ResourceKeySet is a set of resource keys in configuration.
```

Comment:
Please add a bit of context: e.g. this is used to track which resources are added vs updated in the Python mutator.

Reply:
Fixed